n4js-libs/packages/n4js-runtime-fetch/package.json

```diff
@@ -35,6 +35,6 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "n4js-runtime-es2015": "latest"
+    "n4js-runtime-es2015": "^0.1.0"
```

Comment:
I get that these version constraints are only intended to be used locally (lerna will replace them with proper versions).
But why use `^`, why not just `0.1.0` instead of `^0.1.0`?

Reply:
@dbo : I also think similar to @mor-n4 . What do you think?

Reply:
I think it doesn't matter; I assume yarn just checks whether there is a matching version of the depedency in the workspace.

Reply:
use `0.0.1` to align with our eclipse and maven versioning. This is placeholder version that gets replaced when deploying to npm anyway.

Reply:
@dbo wanted it to be 0.1.0. Should we change this to 0.0.1 Daniel?

Reply:
is there technical reason for `0.1.0`? We already have to deal with two versioning schemes, where we use `0.0.1`, so why increase the burden by using different convention here?